### PR TITLE
use freertos max task name length in the logger

### PR DESCRIPTION
**Pull request by Taylorr82**

### CODE REVIEW

File: ex2_system/source/logger/logger.c [hunk 1]
```diff
@@ -27,8 +27,13 @@
...
 #include <redposix.h>
 
+#define XSTR_(X) STR_(X)
+#define STR_(X) #X
+
+
+
 #define DEFAULT_INPUT_QUEUE_LEN 10
-#define TASK_NAME_SIZE 13
+#define TASK_NAME_SIZE configMAX_TASK_NAME_LEN + 3
```
Comment (ArashYzd) [inline ex2_system/source/logger/logger.c:36]: Shouldn't be `configMAX_TASK_NAME_LEN - 3` ?

**Review by wagnerd97 — APPROVED** [review]

File: ex2_system/source/logger/logger.c (hunk 1 above)
Comment (Taylorr82) [reply ex2_system/source/logger/logger.c:36]: The 3 is the brackets and null terminator, so I don't believe so

Comment (ArashYzd) [reply ex2_system/source/logger/logger.c:36]: Ok, I saw the previous value was 13 and configMAX_TASK_NAME_LEN is 16. I really don't know what their values should be :D

**Review by ArashYzd — APPROVED** [review]

File: ex2_system/source/logger/logger.c (hunk 1 above)
Comment (Taylorr82) [reply ex2_system/source/logger/logger.c:39]: Me neither, that's why I made it work with any value

**Pull request merged**